pulsar-client-tools/pom.xml

```diff
@@ -63,6 +63,12 @@
 		<dependency>
 			<groupId>org.asynchttpclient</groupId>
 			<artifactId>async-http-client</artifactId>
+			<exclusions>
+				<exclusion>
+					<groupId>io.netty</groupId>
+					<artifactId>netty-buffer</artifactId>
```

Comment:
Actually, `async-http-client` has dependency on multiple netty submodules (`netty-codec-http`, `netty-transport-epoll`, with a different version than what we're using. 

Since we are already pulling netty-all, we should try to exclude all of them.